bitcoin/CHANGELOG.md

```diff
@@ -1,11 +1,63 @@
-# 0.30 - 2023-XX-XX "The First Crate-Smashing Release"
-
-* Drop `PackedLockTime`, [replace with richer `LockTime` everywhere](https://github.com/rust-bitcoin/rust-bitcoin/pull/1330)
-  Be aware that `LockTime` does not have an `Ord` implementation, so users who need a
-  total ordering on locktimes will be forced to wrap this type. In `Transaction`, which
-  contains a `LockTime` but is `Ord`, we have manually sorted the locktimes based on
-  their consensus encoding. This ordering is somewhat arbitrary -- there is no total
-  ordering on locktimes since they may be measured in either blocks or seconds.
+# 0.30 - 2023-03-19 "The First Crate-Smashing Release"
+
+This release is big, to help users upgrade we wrote a blog post, please see https://rust-bitcoin.org/
+
+- [Import `bitcoin_hashes`](https://github.com/rust-bitcoin/rust-bitcoin/pull/1284) into this repository.
+
+- Update dependencies we control
+  - Depend on the new private crate `bitcoin-private`
+  - Depend on the new `hashes` v0.12
+  - Depend on the new `secp256k1` v0.27
+
+- We moved a lot of modules around, specifically we move basically everything out of `util`.
+
+- API improvements:
```

Comment:
Unsized `Script` is not mentioned here but I think it should since it's quite large and disruptive

Reply:
Added here and also in the "New types" section.